specification/openapi.json

```
@@ -195,24 +195,79 @@
             "type": "integer",
             "description": "Time in ms when difference between in-chat transfer and Tx timestamp considered as acceptable"
           },
-          "minNodeVersion": {
-            "type": "string",
-            "description": "Minimal node API version",
-            "example": "0.8.0"
-          },
           "nodes": {
-            "type": "array",
-            "items": {
-              "$ref": "#/components/schemas/NodeInfo"
-            },
-            "description": "Node links for API"
-          },
-          "services": {
             "type": "object",
```

```suggestion
            "type": "object",
            "required": ["list", "healthCheck"],
```

Make `list` and `healthCheck` props required